homeworks/vitali.timolyanov_XX-NF-XX/homework_7/task_02_login.js

```diff
@@ -0,0 +1,67 @@
+const MAX_FAILED_LOGINS = 3;
+const LOCK_TIME = 10000;
+
+const fs = require('fs');
```

Comment:
You may import any json file with ordinary node require
![image](https://user-images.githubusercontent.com/29049264/50048214-dc40ad80-00ce-11e9-94fa-cdd3f4c2562f.png)

Reply:
Thank you.
I didn't know that node can import something that has no exports.